HackYourFuture-CPH/web-dev-rooms
# Storybook/label

# Description

This PR is for add label/tags in for Mentor Skills like we have on the wireframes. I have done my task to create a label for HTML/CSS.

Fixes # (issue)

# How to test?

I run my code storybook and I can see my label for HTML/CSS. I was having problem to commit my code so, I made some changes in package-lock.json, .eslintrc.js and package.json. 

# Checklist

- [ x ] I have made corresponding changes to the documentation
